components/predict-form/Predict.tsx

```diff
@@ -371,56 +375,96 @@ export function Predict({
                   setQuestionType={setQuestionType}
                 />
 
-                <TextareaAutosize
+                <div
+                  ref={containerRef}
                   className={clsx(
-                    "w-full border-2 border-neutral-300 rounded-md resize-none shadow-lg focus:shadow-xl transition-shadow mb-2",
-                    "focus:outline-indigo-700 placeholder:text-neutral-400",
-                    small
-                      ? "text-md py-2 pl-4 pr-16"
-                      : "text-xl py-4 pl-4 pr-16",
+                    "w-full rounded-md mb-2 relative z-20 border border-neutral-300 bg-neutral-100",
                   )}
-                  placeholder={
-                    placeholder || questionType === QuestionType.MULTIPLE_CHOICE
-                      ? "What day will my project be done by?"
-                      : "Will I finish my project by Friday?"
-                  }
-                  maxRows={15}
-                  onChange={(e) => {
-                    updateTagsPreview(e.currentTarget.value)
-                    void onChangeQuestion(e)
-                  }}
-                  onKeyDown={onEnterSubmit}
-                  defaultValue={questionDefaults?.title}
-                  onMouseDown={(e) => e.stopPropagation()}
-                  {...registerQuestion}
-                  onFocus={() => setQuestionInFocus(true)}
-                  onBlur={() => setQuestionInFocus(false)}
-                  ref={mergeRefs([textAreaRef, formRef])}
-                />
-
-                {showQuestionSuggestionsButton && (
-                  <button
-                    tabIndex={-1}
+                >
+                  <TextareaAutosize
                     className={clsx(
-                      "inline-flex align-middle justify-center text-center btn btn-circle aspect-square absolute right-3 top-[calc(50%+1rem)] hover:opacity-100 !-translate-y-1/2",
-                      showSuggestions ? "btn-active" : "btn-ghost",
-                      !!question && !showSuggestions
-                        ? "opacity-20"
-                        : "opacity-80",
-                      small && "btn-xs px-5",
+                      "w-full resize-none rounded-md p-4 -mb-2 border-b bg-white",
+                      "focus:outline-indigo-700 placeholder:text-neutral-400",
```

Comment:
the outline on this when not focussed is thinner than our other input components, e.g. the resolve by and make prediction inputs below, maybe increase it to match (and so it looks more input-y)? Or could up the outline on the parent div maybe, whatever you think looks best

<img width="705" alt="image" src="https://github.com/user-attachments/assets/ea29a4ac-8952-4c6a-b93c-8be30f336742">